sqlite/comdb2build.c

```diff
@@ -1425,6 +1426,7 @@ void comdb2enableAuth(Parse* pParse, int on)
     arg->auth = auth;
     arg->type = BPFUNC_AUTHENTICATION;
     auth->enabled = on;
+    gbl_check_access_controls = 1;
```

Comment:
what if we fail on master?

Reply:
This is just to check do we need access or not. it will be reset to 0 after next query.
https://github.com/bloomberg/comdb2/blob/master/db/sqlinterfaces.c#L5743